build/configs/rtl8721csm/rdp/defconfig

```diff
@@ -0,0 +1,1114 @@
+CONFIG_USER_ENTRYPOINT="rdp_main"
+CONFIG_BUILTIN_APPS=y
+
+#
+# Examples
+#
+# CONFIG_EXAMPLES_AWS is not set
+# CONFIG_EXAMPLES_EEPROM_TEST is not set
+# CONFIG_EXAMPLES_FOTA_SAMPLE is not set
+# CONFIG_FILESYSTEM_HELPER_ENABLE is not set
+CONFIG_EXAMPLES_RDP=y
+# CONFIG_EXAMPLES_HELLOXX is not set
+# CONFIG_EXAMPLES_IOTBUS_TEST is not set
+# CONFIG_EXAMPLES_KERNEL_SAMPLE is not set
+# CONFIG_EXAMPLES_LIBTUV is not set
+# CONFIG_EXAMPLES_NETTEST is not set
+# CONFIG_EXAMPLES_PROC_TEST is not set
+# CONFIG_EXAMPLES_SELECT_TEST is not set
+# CONFIG_EXAMPLES_SENSORBOARD is not set
+# CONFIG_EXAMPLES_SETJMP_TEST is not set
+# CONFIG_EXAMPLES_SMART is not set
+# CONFIG_EXAMPLES_SMART_TEST is not set
+# CONFIG_EXAMPLES_ST_THINGS is not set
+# CONFIG_EXAMPLES_TESTCASE is not set
+# CONFIG_EXAMPLES_WIFI_TEST is not set
+# CONFIG_EXAMPLES_WORKQUEUE is not set
+CONFIG_EXAMPLES_WIFIMANAGER_TEST=y
+CONFIG_EXAMPLES_MBEDTLS_SELF_TEST=y
+
+#
+# Platform-specific Support
+#
+# CONFIG_PLATFORM_CONFIGDATA is not set
+
+#
+# Shell
+#
+CONFIG_TASH=y
+CONFIG_TASH_MAX_COMMANDS=132
+CONFIG_TASH_MAX_STORE_COMMANDS=10
+# CONFIG_TASH_USLEEP is not set
+# CONFIG_TASH_COMMAND_INTERFACE is not set
+CONFIG_TASH_CMDTASK_STACKSIZE=4096
+CONFIG_TASH_CMDTASK_PRIORITY=100
+# CONFIG_TASH_SCRIPT is not set
+
+#
+# System Libraries and Add-Ons
+#
+# CONFIG_SYSTEM_CLE is not set
+# CONFIG_SYSTEM_CUTERM is not set
+# CONFIG_SYSTEM_FOTA_HAL is not set
+# CONFIG_SYSTEM_I2CTOOL is not set
+# CONFIG_SYSTEM_INIFILE is not set
+CONFIG_SYSTEM_PREAPP_INIT=y
+CONFIG_SYSTEM_PREAPP_STACKSIZE=2048
+CONFIG_SYSTEM_IPERF=y
+CONFIG_IPERF_PRIORITY=100
+CONFIG_IPERF_STACKSIZE=4096
+CONFIG_SYSTEM_NETDB=y
+CONFIG_SYSTEM_NETDB_STACKSIZE=2048
+CONFIG_SYSTEM_NETDB_PRIORITY=100
+# CONFIG_SYSTEM_RAMTEST is not set
+CONFIG_SYSTEM_READLINE=y
+CONFIG_READLINE_ECHO=y
+CONFIG_SYSTEM_INFORMATION=y
+CONFIG_SYSTEM_CMDS=y
+CONFIG_FS_CMDS=y
+CONFIG_FSCMD_BUFFER_LEN=32
+CONFIG_NET_CMDS=y
+CONFIG_NET_PING_CMD=y
+CONFIG_NET_PING_CMD_ICOUNT=5
+CONFIG_ENABLE_DATE=y
+CONFIG_ENABLE_ENV_GET=y
+CONFIG_ENABLE_ENV_SET=y
+CONFIG_ENABLE_ENV_UNSET=y
+CONFIG_ENABLE_FREE=y
+# CONFIG_ENABLE_IRQINFO is not set
+CONFIG_ENABLE_KILL=y
+CONFIG_ENABLE_KILLALL=y
+CONFIG_ENABLE_PS=y
+# CONFIG_ENABLE_STACKMONITOR is not set
+CONFIG_ENABLE_UPTIME=y
+# CONFIG_SYSTEM_VI is not set
+
+#
+# Loadable apps Configuration
+#
+
+#
+# Runtime Environment
+#
+# CONFIG_ENABLE_IOTJS is not set
+
+#
+# Device Management
+#
+
+#
+# Binary manager
+#
+
+#
+# Task Monitor
+#
+
+#
+# Task manager
+#
+# CONFIG_TASK_MANAGER is not set
+
+#
+# Event Loop Framework
+#
+# CONFIG_EVENTLOOP is not set
+
+#
+# Messaging Framework
+#
+# CONFIG_MESSAGING_IPC is not set
+
+#
+# Preference Support
+#
+# CONFIG_PREFERENCE is not set
+
+#
+# Things Management
+#
+
+#
+# IoTBus Framework
+#
+CONFIG_IOTBUS=y
+CONFIG_IOTBUS_GPIO=y
+CONFIG_IOTBUS_UART=y
+
+#
+# Security Framework
+#
+# CONFIG_SECURITY_API is not set
+
+###################################
```

Comment:
This is not from menuconfig, I think. Please add Kconfig these things.

Reply:
done